beast/plotting/plot_cmd.py

```diff
@@ -19,7 +19,7 @@ def plot_cmd(
     mag1_filter="F475W",
     mag2_filter="F814W",
     mag3_filter="F475W",
-    savefig=False,
+    savefig=None,
```

Comment:
Why this change?

Reply:
Ah, the original implementation had the argparse default to True, so it was trying to set True as the file extension (which obviously didn't work).  I can switch this back to a boolean if that's preferred.

Reply:
I think that is the right way to go.  May not matter, but it was working with False before.